src/devices/src/virtio/balloon/persist.rs

```diff
@@ -20,12 +20,92 @@ use crate::virtio::balloon::device::{BalloonStats, ConfigSpace};
 use crate::virtio::persist::VirtioDeviceState;
 use crate::virtio::{DeviceState, Queue};
 
-#[derive(Versionize)]
+#[derive(Clone, Versionize)]
+pub struct BalloonConfigSpaceState {
+    num_pages: u32,
+    actual_pages: u32,
+}
+
+impl BalloonConfigSpaceState {
```

Comment:
I would drop this impl since these fns are used only once.

Reply:
Done.